TESTING.md

```diff
@@ -249,6 +249,54 @@ cargo make watch-test
 
 **Note:** For more information on publishing AMIs see [publishing](PUBLISHING.md).
 
+### vmware-k8s
+
+First, an initial management cluster needs to be created using [`EKS Anywhere`](https://anywhere.eks.amazonaws.com/docs/getting-started/production-environment/vsphere-getstarted/#create-an-initial-cluster).
```

Comment:
Does it have to be with `EKS Anywhere`? Is this the cluster to be tested or the cluster that runs the agents?

Regardless of the above, my understanding was that the OVAs can be used on-prem withouth having to use EKS Anywhere. Is testsys tightly coupled with EKS Anywhere?

Reply:
This is the cluster that has the cluster API CRDs and manages other vSphere K8s cluster objects. This has to be created with EKS anywhere because there are special EKS-A CRDs and modifications to the upstream CAPI CRDs. Currently, to create vSphere K8s clusters with bottlerocket images, you have to use EKS-A.